File: MemeShareJava/app/src/main/java/com/example/memesharekotlin/model/KullaniciResponse.java
package com.example.memesharekotlin.model;

import com.google.gson.annotations.SerializedName;

public class KullaniciResponse {
    @SerializedName("success")
    public boolean success;

    @SerializedName("message")
    public String message;

    @SerializedName("user_id")
    public int userId;

    public KullaniciResponse(boolean success, String message, int userId) {
        this.success = success;
        this.message = message;
        this.userId = userId;
    }
}

File: MemeShareJetpack/app/src/main/java/com/example/memesharejetpack/model/KullaniciResponse.kt
package com.example.memesharejetpack.model

import com.google.gson.annotations.SerializedName

data class KullaniciResponse(
    @SerializedName("success")
    val success: Boolean,
    @SerializedName("message")
    val message: String,
    @SerializedName("user_id")
    val userId: Int
)
